pkg/nas/cloud/nas_client_v2.go

```diff
@@ -32,7 +32,7 @@ func newNasClientV2(region string) (*sdk.Client, error) {
 	// set endpoint
 	ep := os.Getenv("NAS_ENDPOINT")
 	if ep == "" {
-		ep = fmt.Sprintf("nas-vpc.%s.aliyuncs.com", region)
+		ep = GetEndpointForRegion(region)
```

Comment:
Will this fix the issue we met? We have test failure in `cn-wulanchabu`, in which we still use vpc endpoint in the new version.